docs/assets/js/selector.js

```diff
@@ -0,0 +1,32 @@
+function selectOption(s) {
+    if (!s) {
```

Comment:
generally we prefer to avoid abbreviated variable names

Reply:
@NikkiWines  would `select` be a better name?

`s` is a select entity being passed.

```
<select>
  <option>Mobile</option>
  <option>Desktop</option>
</select>
```

Reply:
let me know if you have a better suggestion, thanks!